CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ## Unreleased
 
+### Deprecated features
+
+#### Stop using the `useTudorCrown` parameter in the Heading component
+
+As the GOV.UK logo rollout has ended, the Tudor crown logo is now shown by default and the `useTudorCrown` parameter has been deprecated. We will remove it in the next major release.
```

Comment:
As we're planning on releasing this on the 23rd, I suggest we soften this language.

```suggestion
As the beginning of the GOV.UK logo rollout has past, the Tudor crown logo is now shown by default and the `useTudorCrown` parameter has been deprecated. We will remove it in the next major release.
```

Reply:
Done but for swapping `past` for `passed`, which my spellchecker says is the word that should be there.